### ISSUE TITLE
Standard Formats

### ISSUE THREAD

**Issue by renepickhardt** [renepickhardt/generalized-language-modeling-toolkit#11]
the output of language models and n-grams should follow standard formats e.g.

weighted finite state transducer format (WFST)
ARPA format

currently I am not sure if more formats exist. they should be researched and implemented.

**Comment by lschmelzeisen** [comment]
In our experimentation we found that we won't be able to conform with standard formats (namely APRA) I guess we won't have this for stable release then?

**Comment by renepickhardt** [comment]
yes our current tool will not provide an arpa file since a good decoder is shipped. I would still leave this bug open (maybe move it to a different milestone) since saving our model as an FST might still be an option also for saving space and improving runtime
